internal/android/safetynet_test.go

```diff
@@ -61,7 +44,7 @@ func TestVerifyAttestation(t *testing.T) {
 		t.Fatalf("error verifying attestation %v", err)
 	}
 
-	expectedNonce := NewNonce(publish).Nonce()
+	expectedNonce := "VLIju4jpickw6QDy/poqNlVd/0aIsJlNclAIh2jBz1M="
```

Comment:
I would rather have this calculated - it’s the only piece where we have a payload being validated from the reference APP, and I’ve used this multiple times to verify new changes to the protocol.

Reply:
Done. I didn't do it originally because it requires moving the `VerifyOpts` helper from the APIConfig into the `android` package to resolve a circular import.